CONTRIBUTING.md

```diff
@@ -257,6 +257,16 @@ the node describing the language and `@content` to describe the injection region
 ```
 @{language} ; e.g. @html to describe a html region
 
-@language ; dynamic detection of the injection language (i.e. the text of the captured node describes the language)
-@content ; region for the dynamically detected language
+@language ; dynamic detection of the injection language (i.e. the text of the captured node describes the language).
+@content ; region for the dynamically detected language.
+@combined ; This will combine all matches of a pattern as one single block of content.
+```
+
+### Indents
+
+```
+@indent ; Indent when finding this node
+@branch ; Dedent when finding this node
+@return ; Dedent when finding this node
+@ignore ; Skip this node when calculating the indentation level
```

Comment:
Maybe @kyazdani42 has an input on this too, just to be sure we are exact on the terms ?

Reply:
these are the terms indeed. I'd use matching instead of finding but that's allright this way

Reply:
Changed!